.circleci/config.yml

```diff
@@ -4,37 +4,48 @@ jobs:
     docker:
       - image: linuxbrew/linuxbrew
     environment:
-      CIRCLE_REPOSITORY_URL: https://github.com/linuxbrew/homebrew-xorg
       HOMEBREW_DEVELOPER: 1
       HOMEBREW_NO_AUTO_UPDATE: 1
       HOMEBREW_VERBOSE: 1
       HOMEBREW_VERBOSE_USING_DOTS: 1
     steps:
-      - run: |
-          cd /home/linuxbrew/.linuxbrew/Homebrew
-          git fetch origin --unshallow --tags
-          git reset --hard origin/master
-      - checkout
-      - run: git remote set-url origin $CIRCLE_REPOSITORY_URL
-      - run: if [ -e .git/shallow ]; then echo git fetch --unshallow; fi
-      - run: git fetch origin
-      - run: git config --global user.name LinuxbrewTestBot
-      - run: git config --global user.email testbot@linuxbrew.sh
-      - run: chmod 0644 *.rb
-      - run: mkdir -p /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew
-      - run: cp -a . /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew/$CIRCLE_PROJECT_REPONAME
-      - run: |
+      - run:
+        name: Set up Git
+        command: |
+          git config --global user.name LinuxbrewTestBot
+          git config --global user.email testbot@linuxbrew.sh
+      - run:
+        name: Set up Linuxbrew
+        working_directory: /home/linuxbrew/.linuxbrew/Homebrew
+        no_output_timeout: 5m
+        command: |
+          git pull --ff-only
+      - run:
+        name: Set up Linuxbrew/$CIRCLE_PROJECT_REPONAME
+        no_output_timeout: 5m
```

Comment:
Ditto

Reply:
10 minutes might be an overkill for this step, don't you think?

Reply:
Sure, but I'd rather not clutter up the config with items that won't make a difference.

Reply:
ok, will remove